### ISSUE TITLE
Unable to change boot settings after reboot

### ISSUE THREAD

**Issue by anoother** [ocadotechnology/wry#3]
eg.

First reboot works:

```
>>>  dev.boot.enable('BIOSSetup')
True

>>> dev.power.reset()
True
```

Once the machine is back up, and attempt to apply a new boot configuration fails thus:

```
>>> dev.boot.enable('BIOSSetup')

... [traceback clipped] ...

WSManFault: The Selectors for the resource are not valid.
```

This is probably due to us having a new `CIM_BootService` instance after reboot, and we are assuming there is only one...

**Comment by anoother** [comment]
Fixed in the `boot_control` branch.

Due to be merged in to `master` once the code is cleaned up a little.

**Comment by anoother** [comment]
Fixed by 9731648
